packages/protocol/graphics.ts

```diff
@@ -652,63 +663,48 @@ export async function getFirstMeaningfulPaint(limit: number = 10) {
   }
 }
 
-// precache this many milliseconds
-const precacheTime = 5000;
-// startTime of the currently running precacheScreenshots() call
-let precacheStartTime = -1;
+////////////////////////////////////////////////////////////////////////////////////////////////
+// The callbacks below allow external code to observe specific events.
+// This design pattern is a short term strategy to remove external imports from this package.
+//
+// TODO We should revisit this as part of a larger architectural redesign (#6932).
+////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
I laugh in the face of your 80-character limits 😈

Reply:
🤣

Reply:
Also: https://baymard.com/blog/line-length-readability

Reply:
I don't know how applicable that article is to code/comments. Reading code/comments seems like a pretty different task than reading a website's "body text" to me.

If we think it _is_ applicable then even 80 chars is too long too and we should aim for 50-75.